### PR TITLE
Table styling

### PR DESCRIPTION
**Pull request by eireland**
Adds more thematic styling for menu and modals.

### CODE REVIEW

File: v3/src/components/case-table/insert-cases-modal.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export const InsertCasesModalContent: React.FC<IInsertCasesModalProps> =
 
   return (
     <FormControl display="flex" flexDirection="column">
-      <FormLabel display="flex" flexDirection="row"># cases to insert:
-        <NumberInput size="xs" w="75" min={0} ml={5} defaultValue={1} data-testid="num-case-input"
+      <FormLabel h="20px" display="flex" flexDirection="row" alignItems={"center"}># cases to insert:
```
Comment (kswenson) [inline v3/src/components/case-table/insert-cases-modal.tsx:18]:
```suggestion
      <FormLabel h="20px" display="flex" flexDirection="row" alignItems="center"># cases to insert:
```
Consistency

File: v3/src/components/case-table/case-table.scss [hunk 2]
```diff
@@ -15,28 +15,30 @@ $table-body-font-size: 8pt;
     font-size: $table-body-font-size !important;
     color: #222222;
 
-    .rdg-header-row {
-      line-height: 14px !important;
+    .rdg-row {
+      .rdg-header-row {
```
Comment (kswenson) [inline v3/src/components/case-table/case-table.scss:19]:
```suggestion
    .rdg-header-row {
```
`.rdg-header-row` is not inside `.rdg-row`, so this change simply disables column header formatting.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Generally looks good -- I like the use of Chakra theming. Just a couple of comments, the main one being to remove the undesirable change to column header formatting.

**Review by kswenson — APPROVED** [review]
👍 Changes look good

**Review by bfinzer — APPROVED** [review]
👍🏻LGTM

**Pull request merged**